app/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,375 @@
+    <string name="label_send_btc_create_order">Creando ordine XMR.TO</string>
+
+    <string name="label_send_btc_retry">Tocca per riprovare</string>
+
+    <string name="label_send_txid">ID Transazione</string>
+    <string name="label_send_address">Indirizzo di Destinazione</string>
+    <string name="label_send_payment_id">ID Pagamento</string>
+    <string name="label_send_notes">Note</string>
+
+    <string name="backup_progress">Backup in corso</string>
+    <string name="archive_progress">Archiviazione in corso</string>
+    <string name="rename_progress">Rinomina in corso</string>
+    <string name="open_progress">Controllando connessione con daemon </string>
+
+    <string name="service_progress">Rimettendo le cose a posto &#8230;\nPuò richiedere del tempo!</string>
+
+    <string name="backup_success">Backup effettuato con successo</string>
+    <string name="backup_failed">Backup fallito!</string>
+    <string name="archive_success">Archiviazione effettuata con successo</string>
+    <string name="archive_failed">Archiviazione fallita!</string>
+    <string name="delete_failed">Cancellazione fallita!</string>
+    <string name="rename_failed">Rinomina fallita!</string>
+
+    <string name="label_daemon">Nodo</string>
+    <string name="prompt_daemon">([&lt;utente&gt;:&lt;password&gt;@]&lt;daemon&gt;[:&lt;porta&gt;])</string>
+    <string name="prompt_mainnet">Selezione della rete</string>
+    <string name="connect_stagenet">StageNet</string>
+    <string name="connect_testnet">TestNet</string>
+    <string name="connect_mainnet">MainNet</string>
+    <string name="status_walletlist_loading">Caricando la lista dei portafogli</string>
+    <string name="status_wallet_loading">Caricando portafoglio &#8230;</string>
+    <string name="status_wallet_unloading">Salvando portafoglio</string>
+    <string name="status_wallet_unloaded">Portafoglio salvato</string>
+    <string name="status_wallet_unload_failed">Salvataggio portafoglio fallito!</string>
+    <string name="status_wallet_connecting">In connessione &#8230;</string>
+    <string name="status_wallet_connect_failed">Connessione con il nodo fallita!\nCheck username/password</string>
+    <string name="status_wallet_connect_timeout">Connessione con il nodo scaduta!\nTry again or another.</string>
+    <string name="status_wallet_node_invalid">Nodo invalido!\nProvane un altro.</string>
+    <string name="status_wallet_connect_ioex">Impossibile raggiungere il nodo!\nProva di nuovo o provane un altro.</string>
+
+    <string name="status_working">Lavorandoci su &#8230;</string>
+    <string name="status_wallet_disconnected">Disconnesso</string>
+
+    <string name="status_transaction_sent">Transazione inviata!</string>
+    <string name="status_transaction_failed">Transazione fallita: %1$s</string>
+    <string name="status_transaction_prepare_failed">Impossibile creare la transazione!</string>
+
+    <string name="send_xmrto_timeout">Amico, hai aspettato troppo a lungo!</string>
```

Comment:
Please use gender neutral terminology: `Amico/a`

Reply:
You forgot this :)

Reply:
Agreed.